### ISSUE TITLE
Memory errors in CheckIt, screen corruption in MDA mode

### ISSUE THREAD

**Issue by skiselev** [schlae/graphics-gremlin#17]
The FPGA byte code from November 20, 2021 (the latest) produces memory errors at certain memory addresses in both MDA and CGA modes.
It also results in memory/screen corruption in MDA mode. The location on the screen seem to correspond to these addresses (typically column 64).
The issue is reproducible in several cards built exactly according to the BoM.

Typical error locations:
* CGA: 0xB9000, 0xBA000
* MDA: 0xB0080, 0xB0400, 0xB0800, 0xB0880, 0xB0C80

Different cards might show errors only in some of these locations. Perhaps there is a timing violation that results in memory corruption in for these certain addresses

**Comment by schlae** [comment]
Yes, it's a timing issue, I haven't yet been able to track it down.

**Comment by skiselev** [comment]
Tried rebuilding with Yosys-0.31+1 (oss-cad-suite-linux-x64-20230719.tgz), and I am getting even more memory corruption.
Seems to be every location 0x20'th location
![image](https://github.com/schlae/graphics-gremlin/assets/11326662/7714067e-2815-41c2-9316-26ec7abce1e8)
